srcbook/package.json

```diff
@@ -5,17 +5,19 @@
   "type": "module",
   "bin": "./bin/cli.mjs",
```

Comment:
I think this is what is needed for the binary in production, not the `start` command below. This will need the proper path.

Reply:
@benjreinhart yep totally missed that, fixed it and tested with `npm link` `srcbook start` works